build/media-suite_compile.sh

```diff
@@ -403,6 +403,7 @@ if [[ $ffmpeg != "no" ]] && enabled libzimg &&
     do_checkIfExist
 fi
 
+enabled ladspa && do_pacman_install ladspa-sdk # actually superseeded by lv2 now but better than nothing
```

Comment:
Either make the comment on a separate line, on the commit message, or not at all.